### ISSUE TITLE
When copilot joins a project, he gets notification about it.

### ISSUE THREAD

**Issue by maxceem** [appirio-tech/connect-app#2611]
I joined a project as a copilot and in the notifications center I see notification about this event which looks like some another copilot jointed the project, but it was me:

![image](https://user-images.githubusercontent.com/146016/46802052-d658e400-cd8e-11e8-8c70-0fc4c3808b45.png)

I guess copilot shouldn't get this notification about himself.

What do you think @RishiRajSahu ?

**Comment by vic-topcoder** [comment]
you're right, @maxceem - no notification should be dispatched to the copilot. I should not get notifications for my own activity on the system at all.

**Comment by RishiRajSahu** [comment]
@maxceem shouldn't be the case, I have to check whether we can have role based check in our app but adding that again is a challenge because it will restrict the notification from other members with copilot role in our project to see the notification.

**Comment by RishiRajSahu** [comment]
@maxceem do we have a `userId` in the payload given by notifications service so that we can filter notifications by comparing the logged in `userId` with notification creator `userId` ?

**Comment by maxceem** [comment]
@RishiRajSahu yes, we have it:

![image](https://user-images.githubusercontent.com/146016/46899997-cbf13400-cecd-11e8-9af3-74a3bf88cad5.png)

- `initiatorUserId` user which made the action (manager which added copilot)
- `userId` user which was added (copilot)

I guess we still have to show this notification to users, but change the text. As a copilot I still want to get notifications about being added to a project. Though text should be something like:
`You've been added to project Development Integration as a copilot!`

If we want just to remove it, we can cofugure it in https://github.com/topcoder-platform/tc-notifications/blob/dev/connect/events-config.js by adding `exclude: { toUserHandle: true }` properties to `BUS_API_EVENT.CONNECT.MEMBER.JOINED`, `BUS_API_EVENT.CONNECT.MEMBER.COPILOT_JOINED`, `BUS_API_EVENT.CONNECT.MEMBER.MANAGER_JOINED` events.
